src/Public/TraceContext.cs

```diff
@@ -0,0 +1,54 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Google.Protobuf.Collections;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Functions.PowerShellWorker
+{
+    /// <summary>
+    /// Custom TraceContext constructed from the TraceContext members in gRPC.
+    /// </summary>
+    public class TraceContext
+    {
+        /// <summary>
+        /// Constructor for TraceContext.
+        /// </summary>
+        public TraceContext(string traceParent, string traceState, MapField<string, string> attributes)
+        {
+            Traceparent = traceParent;
+            Tracestate = traceState;
+            Attributes = GetCaseInsensitiveAttributes(attributes);
+        }
+
+        /// <summary>
+        /// Gets the traceparent.
+        /// </summary>
+        public string Traceparent { get; }
```

Comment:
Casing: TraceParent, TraceState

Reply:
Apparently, there is a decision on these names that I was not aware of (https://github.com/Azure/azure-functions-host/pull/4917#discussion_r324471617), so please ignore my comment here.